### ISSUE TITLE
plugin.apply is not a function

### ISSUE THREAD

**Issue by dungla4** [ankane/vue-chartkick#3]
There's an issue "plugin.apply is not a function" when I tried to use vue-chartkick. Please help
This is the packages version installed from npm:
- chartkick@2.2.3
- vue-chartkick@0.1.3
- highcharts@5.0.9

The js file contains:
`Vue = require('Vue')`
`VueChartkick = require('vue-chartkick')`

`Vue.use(VueChartkick)`

**Comment by ankane** [comment]
Hey @dungla4, `VueChartkick` is a component rather than a plugin, so no need to call `Vue.use`.

**Comment by dungla4** [comment]
@ankane Thank you very much. In the main js file I put `require('vue-chartkick')` to make sure this your lib loaded. However I get the error

> [Vue warn]: You are using the runtime-only build of Vue where the template option is not available. Either pre-compile the templates into render functions, or use the compiler-included build. 
(found in component <line-chart>)

My vue file is below:
`<template>
  <line-chart :data="chartData"></line-chart>
</template>
<script>
  chartData: [["Jan", 4], ["Feb", 2], ["Mar", 10], ["Apr", 5], ["May", 3]]
</script>
`

I also read the guide `https://vuejs.org/v2/guide/components.html#Registration` and put require your lib before initiate My component but no luck. Can you please take a look?

**Comment by dungla4** [comment]
Hey @ankane , seem that register global component has conflict with `Vue Router`. I try to create local component based on your code and it works.
Can you solve this issue? Or can you port your lib to plugin, it also works with `Vue Router` (e.x `vue-highcharts`)

**Comment by ankane** [comment]
Hey @dungla4, I've tried converting it to plugin on master. You should be able to do:

```js
var Chartkick = require("chartkick")
var VueChartkick = require("vue-chartkick")

Vue.use(VueChartkick, {Chartkick: Chartkick})
```

Let me know how it goes.

**Comment by dungla4** [comment]
@ankane How can I test it? I still get the error `plugin.apply is not a function`. The version is here:
├── chartkick@2.2.3
└── vue-chartkick@0.1.3

**Comment by ankane** [comment]
Hey @dungla4, you'll need to specify the dependency from Github. https://docs.npmjs.com/files/package.json#github-urls

**Comment by ankane** [comment]
Tested and released 0.2.0.
